src/snowflake/snowpark/modin/plugin/extensions/index.py

```diff
@@ -2625,6 +2626,7 @@ def array(self) -> ExtensionArray:
         """
         return the array of values
         """
+        WarningMessage.index_to_pandas_warning("array")
```

Comment:
this does not need a warning. This is expected.

Reply:
Can you help to remove the warning message for `str` too?

Reply:
done!